### DISCUSSION TITLE
Changing default audio output on 5 Plus

### DISCUSSION THREAD

**Discussion by rawsome1** [Joshua-Riek/ubuntu-rockchip#470] [General]
Is there a way to change the default audio output and make it stick on reboot?

**Comment by bagong** [comment]
If you switch to full pipewire-use (instead of pulseaudio), it will stick. The todo's on the web how to switch to pipewire worked for me.

**Reply by Eternalightwithin** [reply]
This is GREAT Bagong.  Just what I needed to hear.  Any particular website? It's been a year since I ripped out pulse on Ubuntu 22.x and levered in Pipewire. lol, this was before pipewire hit ver. 1.0

**Reply by bagong** [reply]
I think I used this one ;-)
https://ubuntuhandbook.org/index.php/2022/04/pipewire-replace-pulseaudio-ubuntu-2204/
